### PR TITLE
Updated export_players information export

### PR DESCRIPTION
**Pull request by michael1241**
Added 'date_created' and 'friends' output in the export_player page.

str() called on date_created to make it JSON serialisable.

### CODE REVIEW

File: heltour/tournament/admin.py [hunk 1]
```diff
@@ -661,7 +661,9 @@ def player_info_view(self, request, object_id, player_name):
             'league': season.league,
             'player': season_player.player,
             'reg': reg,
-            'has_played_20_games': has_played_20_games
+            'has_played_20_games': has_played_20_games,
+            'date_created': str(sp.registration.date_created or sp.player.date_created),
+            'friends': sp.registration.friends or None
```
Comment (lakinwecker) [inline heltour/tournament/admin.py]: if registration is None both of these lines will fail. I would suggest using this format:

`sp.registration.friends if sp.registration else None`
and
`(sp.registration.date_created if sp.registration else sp.date_created).isoformat()`\

This also uses the `isoformat()`

Comment (lakinwecker) [inline heltour/tournament/admin.py]: This also uses the date_created on the seasonplayer directly, not on the Player record which might be quite old.

File: heltour/tournament/admin.py [hunk 2]
```diff
@@ -661,7 +661,9 @@ def player_info_view(self, request, object_id, player_name):
             'league': season.league,
             'player': season_player.player,
             'reg': reg,
-            'has_played_20_games': has_played_20_games
+            'has_played_20_games': has_played_20_games,
+            'date_created': sp.registration.friends if sp.registration else None,
+	    'friends': (sp.registration.date_created if sp.registration else sp.date_created).isoformat()
```
Comment (lakinwecker) [inline heltour/tournament/admin.py]: Looks better, but I think there is a tab mixed in with the spaces here.  Can you verify that you're using spaces only for indentation? 4 spaces per level of indentation is what is proposed by PEP-8 (the python formatting guidelines) and is what we use in this project.

File: heltour/tournament/admin.py [hunk 3]
```diff
@@ -661,7 +661,9 @@ def player_info_view(self, request, object_id, player_name):
             'league': season.league,
             'player': season_player.player,
             'reg': reg,
-            'has_played_20_games': has_played_20_games
+            'has_played_20_games': has_played_20_games,
+            'date_created': sp.registration.friends if sp.registration else None,
```
Comment (cyanfish) [inline heltour/tournament/admin.py]: `sp` is not defined here, you'll either need to expand it to `season_player` or rename the variable above

Comment (cyanfish) [inline heltour/tournament/admin.py]: Also, I think this is the wrong spot to add this. You probably want line 610, in which case `sp` would be defined.

**Pull request merged**